docs/deployment/choosing_configuring_rds.md

```diff
@@ -41,6 +41,12 @@ will be able to handle the spikes in your database load.
 
 ## General Configuration Guidelines
 
+### Aurora Serverless V2 Capacity Range
+
+Serverless V2 allows users to configure the minumum and maximum number of ACUs the custer should use.
```

Comment:
See below comment. We could at some point make recommendations here but I don't think we have them now.